packages/grid/x-data-grid/src/tests/keyboard.DataGrid.test.tsx

```diff
@@ -716,13 +716,13 @@ describe('<DataGrid /> - Keyboard', () => {
         <DataGrid rows={rows} columns={columns} />
       </div>,
     );
-    expect(renderCell.callCount).to.equal(2);
+    expect(renderCell.callCount).to.equal(6);
```

Comment:
And the number of rerenders changed here again 🤷🏻‍♂️
https://github.com/mui/mui-x/pull/7155#discussion_r1044772210